### PR TITLE
updating dockerfile to include build time arguments from TC and passi…

### PR DESCRIPTION
**Pull request by P0NDER0SA**
…ng them to the app as environment variables so we can display or include build time stamps and Team City build information where necessary.

# Description

LJ-40

1-3 sentence description of the feature you added.

sends in TC_BUILD and BUILD_DATE as environment variables to the application..
To test, check the metadata on the home page after the application gets deployed to our dev cloud url:
https://lifejourneys-main.dev.dts-stn.com/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,5 +29,7 @@ COPY --from=build /build/package*.json ./
 COPY --from=build /build/.next ./.next
 COPY --from=build /build/public ./public
 RUN npm install next
+RUN echo $BUILD_DATE 
+RUN echo $TC_BUILD
```
Comment (pcraig3) [inline Dockerfile:33]: We could remove this, right? All the same, doesn't hurt to keep it.

File: components/layout.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 export default function Layout({ children, title }) {
   return (
     <div className="flex flex-col items-center min-h-screen py-2">
       <Head>
         <link rel="icon" href="/images/ace.jpg" />
+        <meta
+          name="og:builddate"
+          content={process.env.NEXT_PUBLIC_BUILD_DATE}
+        />
+        <meta name="og:buildid" content={process.env.NEXT_PUBLIC_TC_BUILD} />
```
Comment (pcraig3) [inline components/layout.js:13]: What happens if these aren't present?

File: next.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  env: {
+    NEXT_PUBLIC_BUILD_DATE: process.env.NEXT_PUBLIC_BUILD_DATE,
+    NEXT_PUBLIC_TC_BUILD: process.env.NEXT_PUBLIC_TC_BUILD,
```
Comment (pcraig3) [inline next.config.js:4]: Is it possible to set defaults here or does it matter?

Comment (MarcoGoC) [reply next.config.js:4]: we need to add 2 variables to the env file :)

Comment (P0NDER0SA) [reply next.config.js:4]: you can add the variables (or not) it doesn't break or do anything if they dont exist, these variables are mainly useful for deployed builds to see the build date and TC build id in the source at a quick reference. :)

File: Dockerfile (hunk 1 above)
Comment (P0NDER0SA) [reply Dockerfile]: gonna keep em in for now (helps for debugging, but could remove them in the future for sure).

**Review by pcraig3 — APPROVED** [review]
Seems good to me.

**Pull request merged**